src/Controls/src/Core/Platform/ShapeExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.Maui.Controls.Shapes;
+using Microsoft.Maui.Graphics;
+
+namespace Microsoft.Maui.Controls.Platform
+{
+	internal static class ShapeExtensions
+	{
+		internal static WindingMode GetPathWindingMode(this IDrawable drawable, IShapeView shapeView)
+		{
+			if (drawable is not ShapeDrawable || shapeView is null || shapeView.Shape is not Path path)
```

Comment:
did you mean `shapeView is not null`?

```suggestion
			if (drawable is not ShapeDrawable || shapeView is not null || shapeView.Shape is not Path path)
```

Reply:
Is the correct value. Is the drawable is not a Shape, or ShapeView is null, return the default WindingMode.